extension.js

```diff
@@ -161,8 +161,9 @@ function openSSHConnection(serverName, isFastConnection, forwardingArgs = null)
             terminal = vscode.window.createTerminal(serverName + ((forwardingArgs != null) ? " (Forwarding)" : ""));
             // If custom commands defined send it to terminal
             terminals.push({ "name": serverName, "username": server.configuration.username, "host": server.configuration.host, "terminal": terminal, "isForwarding": (forwardingArgs != null) });
-            if (server.configuration.portKnockingPort !== undefined && server.configuration.portKnockingPort > 0) {
-                terminal.sendText("curl "+server.configuration.host+":"+server.configuration.portKnockingPort);
+            if (server.configuration.portKnocking.port !== undefined && server.configuration.portKnocking.port > 0) {
```

Comment:
Before checking `server.configuration.portKnocking.port` to undefined, add `server.configuration.portKnocking !== undefined`, for preventing undefined property error